codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/protocol/ProtocolTestGenerator.kt

```diff
@@ -166,33 +200,56 @@ class ProtocolTestGenerator(
             writable {
                 val customizations = codegenContext.rootDecorator.endpointCustomizations(codegenContext)
                 params.getObjectMember("builtInParams").orNull()?.members?.forEach { (name, value) ->
-                    customizations.firstNotNullOf { it.setBuiltInOnServiceConfig(name.value, value, "builder") }(this)
+                    customizations.firstNotNullOf {
+                        it.setBuiltInOnServiceConfig(name.value, value, "config_builder")
+                    }(this)
                 }
```

Comment:
this code should be handling the region already

Reply:
Ahhh, I see why it wasn't working now. The `ApiGatewayAccept` test doesn't have endpoint parameters, so this short-circuits and doesn't set a region for that test specifically. I guess I just need to add a default. I don't think there's an easy way to customize the trait to only add endpoint parameters while preserving the rest.